frontend/mulighetsrommet-veileder-flate/src/components/tabell/TiltakstypeTabell.tsx

```diff
@@ -65,39 +92,41 @@ const TiltakstypeTabell = ({ tiltaksgjennomforingsliste }: TiltakstypelisteProps
         <Table.Header>
           <Table.Row>
             <Table.ColumnHeader
-              sortKey="navn"
+              sortKey="title"
               sortable
               className="tabell__kolonne__tiltaksnavn"
               data-testid="tabellheader_tiltaksnavn"
             >
               Tiltaksnavn
             </Table.ColumnHeader>
-            <Table.ColumnHeader sortKey="id" sortable className="tabell__kolonne__tiltaksnummer">
+            <Table.ColumnHeader sortKey="tiltaksnummer" sortable className="tabell__kolonne__tiltaksnummer">
               Tiltaksnr.
             </Table.ColumnHeader>
-            <Table.ColumnHeader sortKey="tiltakskode" sortable className="tabell__kolonne__tiltakstype">
+            {/*TODO fiks sortering*/}
+            <Table.ColumnHeader sortKey="tiltakstype" sortable className="tabell__kolonne__tiltakstype">
               Tiltakstype
             </Table.ColumnHeader>
             <Table.ColumnHeader sortKey="lokasjon" sortable className="tabell__kolonne__oppstart">
               Lokasjon
             </Table.ColumnHeader>
-            <Table.ColumnHeader sortKey="fraDato" sortable className="tabell__kolonne__oppstart">
+            <Table.ColumnHeader sortKey="oppstartsdato" sortable className="tabell__kolonne__oppstart">
               Oppstartsdato
             </Table.ColumnHeader>
+            {/*TODO fiks sortering*/}
             <Table.ColumnHeader sortKey="status" sortable className="tabell__kolonne__plasser">
               Status
             </Table.ColumnHeader>
           </Table.Row>
         </Table.Header>
         <Table.Body>
-          {tiltaksgjennomforingsliste.length === 0 ? (
+          {tiltaksgjennomforing.length === 0 ? (
             <Table.DataCell colSpan={5}>
               <Alert variant="info" className="tabell__alert">
                 Det finnes ingen tiltakstyper med dette søket.
               </Alert>
             </Table.DataCell>
           ) : (
-            tiltaksgjennomforingsliste
+            tiltaksgjennomforing
               .slice()
```

Comment:
Hva gjør denne ekstra `slice()`-metoden her? Er det for å lage en kopi av `tiltaksgjennomforing`? 

Og siden det er en liste, men du ikke vil bruke liste i variabelnavnet, hva med `tiltaksgjennomforinger`, siden det er flertall.